exercises/00-init/src/modules/root/components/users-list.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+class UsersList extends React.Component {
+    constructor() {
+        super();
+        this.state = { 
+            users : []
+        }
+    }
+
+    render() {
+        return <div>
+            <button onClick={ () => this.addUser('Arya', 'Stark')}>Add Arya</button>
+            <button onClick={ () => this.addUser('Daenerys', 'Targaryen')}>Add Daenerys</button>
+            <br/>
+            {this.state.users.length === 0 ? 
+                "No users" :

```

Comment:
Very small note: I would recommend formatting the ternary operator like this:
```
condition
  ? case1
  : case2
```
as opposed to 
```
condition ?
  case1 :
  case2
```
since with the symbols on the beginning of the line, it is more readable, especially if you nest multiple operators together.

Reply:
Agreed, it looks better.